src/components/FAQSection/QuestionFAQ/QuestionFAQ.styles.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+import { Icon } from "@/components/Icon/Icon";
+import styled from "styled-components";
+
+export const SubSectionFAQ = styled.p`
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  transition: 100ms;
+  padding: 0 1rem;
+  margin: 0;
+`;
+
+export const IconWrapper = styled(Icon)`
+  color: ${({ theme }) => theme.colors.black};
+  transition: 100ms linear;
+  flex-shrink: 0;
+`;
+
+export const Summary = styled.summary`
+  display: flex;
+  justify-content: space-between;
+  font-weight: bold;
+  cursor: pointer;
+  border-radius: 4px;
+  padding: 1rem;
+  align-items: center;
+`;
+
+export const QuestionContainer = styled.details`
+  display: flex;
+  width: 100%;
+  padding: 1rem 0;
+  margin: 1rem;
```

Comment:
```suggestion
```

Reply:
don't you think it's better this way? cause by completely removing the margin, the elements ended up too close to each other
```suggestion
  margin: 0.25rem;
```

Reply:
I suggested this and forgot to suggest adding a larget gap on `FAQContainer`, sorry, I would remove the margin from this inner element and add a gap on the outer element - the FAQContainer.